code/modules/mob/living/simple_mob/subtypes/glamour/homunculus.dm

```diff
@@ -33,3 +33,92 @@
 /mob/living/simple_mob/homunculus/update_icons()
 	return
 
+/mob/living/simple_mob/homunculus/evil
+	var/seen_melee
+	var/seen_gun
+	var/transformed
+	var/seen_armour
+	maxHealth = 100
+	health = 100
+
+	faction = "glamour"
```

Comment:
Do not bring back local factions. Define them for an overview

Reply:
Done! (Probably, if you wanna double check, lol)